src/components/Invoice-list.tsx

```diff
@@ -0,0 +1,64 @@
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import EditIcon from '@mui/icons-material/Edit';
+import DeleteIcon from '@mui/icons-material/Delete';
+import IconButton from '@mui/material/IconButton';
+import { Grid } from '@mui/material';
+
+function createData(invoiceNumber: string, created: string, validUntil: string, amount: number, actions: null) {
+  return { invoiceNumber, created, validUntil, amount, actions };
+}
+
+const rows = [
+  createData('21/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('22/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('23/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('24/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('25/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+];
+
+export default function InvoiceList() {
+  return (
+    // Skąd lepiej importować? Bardziej czy mniej szczegółowo?
```

Comment:
Generally it will be slightly more performant, when using default imports, but it's not a big deal. I would rather try to be consistent. You can read more here:
https://mui.com/material-ui/guides/minimizing-bundle-size/#when-and-how-to-use-tree-shaking

Reply:
some default imports not work e.g for adapter js.